hkmc2/shared/src/test/mlscript-compile/apps/parsing-web-demo/main.mls

```diff
@@ -78,9 +43,10 @@ editor.addEventListener of "keydown", event =>
     set editor.selectionEnd = start + 2
     set editor.selectionStart = editor.selectionEnd
 
-selector.addEventListener of "change", event => if selector.value is
-  "example-1" do showExample1()
-  "example-2" do showExample2()
+selector.addEventListener of "change", event =>
+  if examples.get(selector.value) is
+    Some(example) do set editor.value = example.source
+    None do print("Example \"" + selector.value + "\" not found")
```

Comment:
Shouldn't we throw an exception so that it appears as an error in the web console?